src/Task/Base/ParallelExec.php

```diff
@@ -127,6 +146,9 @@ public function run()
             $process->setTimeout($this->timeout);
             $process->start();
             $this->printTaskInfo($process->getCommandLine());
+            if ($this->waitInterval) {
+                sleep($this->waitInterval);
```

Comment:
Record the current time at the beginning of the while loop as the 'next start time'. Inside the while loop, if the 'next start time' <= the current time, and the process queue is empty, then pull an item off of the process queue and start it. Add the wait interval to the 'next start time'.